### PR TITLE
[IMP] stock_location_limit_product: Use product uom

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by nelsonramirezs — APPROVED** [review]

**Review by nikitavaghela — APPROVED** [review]

**Review by bodedra — APPROVED** [review]

File: stock_location_limit_product/models/stock_location_limit.py [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
     _rec_name = 'product_id'
 
-    @api.onchange('product_id')
-    def onchange_uom_id(self):
-        self.uom_id = self.product_id.uom_id.id
-
-    @api.constrains('product_id', 'uom_id')
-    def check_uom_id(self):
-        if self.uom_id.category_id != self.product_id.uom_id.category_id:
-            raise ValidationError(_(
```
Comment (rousseldenis) [inline stock_location_limit_product/models/stock_location_limit.py:21]: @max3903 Don't forget to remove test for this as travis is failing.

File: stock_location_limit_product/models/stock_location_limit.py [hunk 2]
```diff
@@ -1,35 +1,22 @@
...
     _sql_constraints = [
         ('product_uniq', 'unique(product_id,location_id)',
          'You cannot set 2 limits with the same product for a location!')]
 
     product_id = fields.Many2one('product.product', string='Product')
     qty = fields.Float('Maximum Quantity',
                        digits=dp.get_precision('Product Quantity'))
-    uom_id = fields.Many2one('uom.uom', string='UoM')
+    uom_id = fields.Many2one(
+        'uom.uom', related='product_id.uom_id', string='UoM', store=True)
```
Comment (rousseldenis) [inline stock_location_limit_product/models/stock_location_limit.py:21]: :smile: related to https://github.com/OCA/stock-logistics-warehouse/pull/787#discussion_r358095879.

**Review by rousseldenis — APPROVED** [review]
Code review. Just a fix in test

File: stock_location_limit_product/models/stock_location_limit.py (hunk 2 above)
Comment (max3903) [reply stock_location_limit_product/models/stock_location_limit.py:21]: :smile: :+1:

**Pull request merged**